### DISCUSSION TITLE
How to upgrade remote site? (When using rustdesk to access them)

### DISCUSSION THREAD

**Discussion by jamiegau** [rustdesk/rustdesk#7063] [Q&A]
How do you upgrade a remote site when you're using Rustdesk to access the site?
Can you upgrade rustdesk to a newer version while using rustdesk to access the site?

What is the process for Win, Mac, Linux?

Thanks.

**Comment by pixelstuff** [comment]
I'm wondering the same thing. The main app window for the Rustdesk seems to be grayed out when you are remotely logged in.

It would be nice to upgrade from a remote login without needing to maintain a second remote management app on every system so you can log into the second one to update Rustdesk.
